.github/labeler.yml

```diff
@@ -2,35 +2,33 @@ version: v1
 labels:

```

Comment:
i like the regex "i" more
sometimes, someone is using "readme" or "README" or "Readme" .....
"i" is simpler